docs/components/input/README.md

```diff
@@ -169,7 +169,7 @@
                 "name": "size",
                 "type": "string",
                 "default": "medium",
-                "description": "Sets the input field size. Possible values: {  ‘medium’  |  ‘large’  }"
+                "description": "Sets the input field size. Possible sizes are: {  ‘medium’  |  ‘large’  }. Also works with responsive breakpoints. Example: { 'large@xs' }"
```

Comment:
I like how you have the size description in button:  ```"Sets the button size; values can target responsive breakpoints. Breakpoint values are: xs, sm, md, and lg. Examples: { 'small' | 'medium' | 'large' | 'large@sm' }"```
Maybe change this to be more consistent?